cli/reportgen.sh

```diff
@@ -11,6 +12,9 @@ export ADERYN_SKIP_BUILD=1
 # Basic report.md 
 cargo run -- ./tests/contract-playground &
 
+# Basic report.profile.md 
+FOUNDRY_SAMPLE=sample cargo run -- -o report.sample_profile.md ./tests/contract-playground &
+
 # Create report-config.md based on config file
```

Comment:
Automatically generate report for the `sample` FOUNDRY_PROFILE. 

Note: At the moment it may be same in content as `report.md` but it need not be in the future as we reserve the right to change the `src` for the `sample` profile in `foundy.toml` later sometime. 

Let's not link both the reports : )